Feature/sc 3672/tool improvements

vvainola opened the pull request:
## Jira Story
https://epcpower.atlassian.net/browse/SC-3672

## About
- Remove
  - original_frame_name
  - original_multiplexer_name
  - original_signal_name
- Add tool_attributes. See TCU or BCU PR for the use cases.

## Associated Pull Requests

* [ ] https://github.com/epcpower/m-pm/pull/19
* [ ] https://github.com/epcpower/m-tcu/pull/300
* [ ] https://github.com/epcpower/m-bcu/pull/36

## Reproduction Steps

## Validation

vvainola committed:
refactor(SC-3672): Remove unused original_*_name fields

vvainola committed:
feat(SC-3672): Swap unused type_name to decoded_name

vvainola committed:
feat(SC-3672): Add tool_attributes to parameter

vvainola committed:
refactor(SC-3672): Remove unused "Label" field from sunspec enum

Takes space as an column

vvainola committed:
refactor(SC-3672): Remove named_enumerators column

Always true and not used for anything anymore

davidpmichaud commented:
Let's talk about how this decoded_name is going to be used in the call tomorrow.

Also, I'm always amazed at how fast you are... :-)

vvainola committed:
refactor(SC-3672): Remove abbreviation field from parameters

With decoded name field added there would be 4 different
name, abbrevation, decoded_name and CAN signal
possible names so remove the abbreviation which is not
currently used for anything and most of the existing ones
are quite cryptic

Sunspec enumerators still have abbreviation because it's used
for anomaly generation
